wiki/Article_Styling_Criteria/en.md

```diff
@@ -141,42 +176,88 @@ The osu!taiko Champion title goes to [\_yu68](https://osu.ppy.sh/users/6170507).
 
 The escaping character in Markdown is the backslash (`\`). To escape the escape character for a literal backslash, use `\\`.
 
-The title of an article must never be escaped; otherwise, the backslashes will show up on the website.
+The title of an article must never be escaped, as titles are parsed as raw text, not Markdown.
 
-### Emphasizing
+#### Empty lines
 
-*Note: Keep in mind that the misuse and abuse of emphasis will reduce its effectiveness!*
+An empty line between two bodies of text is parsed as two separate paragraphs. Adding more empty lines afterwards will not affect spacing of the paragraphs; it will be parsed as if they were one empty line.
 
-When trying to emphasize words, **never use all capital letters**. Instead, bold the words you are trying to emphasize. For example:
+If an empty line is not present between two lines, the two paragraphs will merge into one. Doing this is not allowed in the osu! wiki.
+
+#### Whitespace
+
+Inserting two trailing spaces at the very end of a line is parsed as a line break. Line breaks are not allowed in the osu! wiki.
+
+### Hatnotes
+
+Hatnotes are short notes placed at the top of an article that help readers navigate to related articles. Hatnotes must be italised, never end with a full stop (`.`), and must be placed immediately after the title or heading.
+
+#### Main page
+
+Main page hatnotes directs a reader to an article with more information. These should be placed after a heading, but before the summary. For example:
 
 ```
-Do not:
-You are NOT allowed to post R-18 skins on the osu! forums.
+## Modding
 
-Do:
-**You are not allowed to post R-18 skins on the osu! forums.**
+*Main page: [Modding](/wiki/Modding)*
+
+Modding is the process of users reviewing (or commonly called "modders") a creator's beatmap in the pending (or work in progress/help) stage.
 ```
 
-#### Bold
+#### See also
 
-Use bold when you need to caution or note something's importance. When bolding, try to avoid bolding single word(s) but instead bold phrases, as it might be hard to notice the bolding on a single word.
+See also hatnotes are used to suggest a reader other points of interest from a given article. For example:
 
-#### Italics
+```
+## Beatmapping
 
-Use italics only when naming a video game or the name of a work.
+*See also: [Storyboarding](/wiki/Storyboarding)*
 
----
+Beatmapping is the process of a creator creating a beatmap. This process includes: selecting music, timing and mapping the beatmap, and testing the beatmap. 
+```
+
+#### For see
 
-You must use italics when writing a note, and for main page and see also links. For example:
+For see hatnotes are similar to see also, but may help reduce ambiguation and usually are more descriptive. For example:
```

Comment:
Note that there is a colon(`:`) after `see`.

Reply:
Hmm, it should be written for each one then too I guess.
Will note how to write them for each one then.